data/json/monsters/zed_misc.json

```diff
@@ -836,6 +836,55 @@
     "upgrades": { "half_life": 20, "into": "mon_zombie_hunter" },
     "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_regenerate", 
+    "type": "MONSTER",
+    "name": { "str": "regenerating zombie" },
+    "description": "A hairless zombie; its pale pinkish flesh appears to be squirming and moving across its body, quickly covering and fixing any wounds that it receives.  Its clothing is riddled with holes, suggesting someone else tried to take it down before you.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 120,
```

Comment:
I'd suggest giving this zombie much lower hp and no armour, so that it's relatively easy to kill with fast or devastating attacks, but cannot be harried and left alone.